README.md

```diff
@@ -97,7 +98,53 @@ sudo apt install libwebkit2gtk-4.1-dev
 sudo dnf install gtk3-devel webkit2gtk4.1-devel
 ```
 
-Fedora does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
+> [!WARNING]
+> Fedora does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
+
+### Nix & NixOS
+
+ ```Nix
+ let
+    # Unstable Channel | Rolling Release
+    pkgs = import (fetchTarball("channel:nixpkgs-unstable")) { };
+    packages = with pkgs; [
+      pkg-config
+      webkitgtk_4_1
+      libayatana-appindicator
+      libappindicator-gtk3
+    ];
+  in
+  pkgs.mkShell {
+    buildInputs = packages;
+  }
+ ```
+
+ > [!TIP]
+ > A `shell.nix` is included in the subdirectory `declerative-shells/etc/shells/nix`. Use as follows:
+ ```sh
+ nix-shell shell.nix
+ ```
+
+#### GUIX
+
+ ```scheme
+ (specifications->manifest
+   '("pkg-config"                ; Helper tool used when compiling
+     "webkitgtk"                 ; Web content engine fot GTK+
+     "libappindicator"           ; Menu in Menu bar
+     "gtk"
+  ))
+ ```
+
+ > [!TIP]
+ > A `manifest.scm` is included in the subdirectory `declerative-shells/etc/shells/guix`. Use as follows:
+ 
+```sh
+ guix shell -m manifest.scm
+ ```
+
+ > [!WARNING]
+ > Guix does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
```

Comment:
This is not needed and the submodule as well, the instructions at Line 104 are enough